tests/sources/fixtures/sharepoint/fixture.py

```diff
@@ -107,21 +109,27 @@ def get_sites(site_collections):
     Returns:
         sites (str): Path of server site
     """
-    args = request.args
+    skip = request.args.get("$skip")
     sites = {"value": []}
-    if request.path == "/sites/site1/_api/web/webs":
+    if "/sites/site1" in request.path:
         return sites
-    elif args.get("$skip"):
-        sites["value"].append(
-            {
-                "Created": "2023-01-30T10:02:39",
-                "Id": adjust_document_id_size("sites-0"),
-                "LastItemModifiedDate": "2023-02-16T06:48:30Z",
-                "ServerRelativeUrl": "/sites/site1",
-                "Title": "site1",
-                "Url": f"http://127.0.0.1:8000/sites/{site_collections}/site1",
-            }
-        )
+    if skip:
+        skip_start = int(skip)
+        if skip_start + 100 >= total_subsites:
```

Comment:
I'm not sure I understand how this works

Reply:
We've used this condition for handling pagination so that if pagination limit is 100 and number of documents we are passing is 150 then in that case this will handle the second api call for the remaining 50 documents.

Reply:
What it the pagination limit is changed? Shouldn't it be picked up from the URL arguments?